src/app/tools/intertwine.cpp

```diff
@@ -69,7 +69,8 @@ void Intertwine::doPointshapeStrokePt(const Stroke::Pt& pt, ToolLoop* loop)
     main_stroke.addPoint(pt);
 
     Strokes strokes;
-    symmetry->generateStrokes(main_stroke, strokes, loop);
+    DynamicsOptions dynamics = loop->getDynamics();
+    symmetry->generateStrokes(main_stroke, strokes, loop, &dynamics);
```

Comment:
As the `generateStrokes()` already receives the `loop`, we can get the dynamics inside `generateStrokes()` instead of adding a new parameter.